deploy/crds/aws_v1alpha1_awsfederatedrole_networkmgmt_cr.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: aws.managed.openshift.io/v1alpha1
 kind: AWSFederatedRole
 metadata:
-  name: customer-admin
+  name: network-mgmt
   namespace: aws-account-operator
 spec:
-  roleDisplayName: customer-admin
-  roleDescription: This is the description of customer-admin
+  roleDisplayName: Network Management
+  roleDescription: User can view all AWS resources, and can edit network-related resoruces to enable things like VPN, VPC, peering, etc.
```

Comment:
"User can view all AWS resources, and can edit VPC related resources to enable services such as IPsec VPN and VPC peering."